content/applications/finance/payment_acquirers/razorpay.rst

```diff
@@ -0,0 +1,43 @@
+===========
+Razorpay
+===========
+
+`Razorpay <https://razorpay.com/>`_ is an online payments provider established in India and
+covering 100+ payment methods.
+
+.. _payment_acquirers/razorpay/configure_dashboard:
+
+Configuration on Razorpay Dashboard
+======================================
+
+#. Log into `Razorpay Dashboard <https://dashboard.razorpay.com/>`_ and go to
+   :menuselection:`Settings --> API Keys`. Generate the new keys and copy the values of the :guilabel:`Key Id` and
```

Comment:
Although the Content Guidelines state 100th character-limit, it is actually a 70-character limit. The sentence should fit within the limit (indicated by the vertical grey line in the screenshot).

![70th character](https://user-images.githubusercontent.com/104567387/174621121-68d2cfe1-2514-4682-8ce9-3f61aad57732.png)